charts/forms-flow-analytics/values.yaml

```diff
@@ -46,16 +47,36 @@ redash:
   multiorg: "false"
   image:
     registry: docker.io #registry to use
-    repository: formsflow/redash # repoisitory for redash image  
-    tag: 10.1.5 # tag of image being used
+    repository: formsflow/redash # repoisitory for redash image
+    tag: 24.04.0 # tag of image being used
+    pullPolicy: Always
   database:
     password: postgres # admin password
     url: "postgresql://postgres:postgres@forms-flow-analytics-postgresql:5432/postgres" # redash database url - i.e forms-flow-analytics-postgresql
 
 resources:
   limits:
-    cpu: 200m
+    cpu: 2000m
```

Comment:
Can you try lowering this and see if it works? It's huge jump from .2c to 2c
Same for requests too